style.css

```diff
@@ -0,0 +1,376 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+body {
+  font-family: 'Inter', sans-serif;
+  display: flex;
+  align-content: center;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-color: #e5e5e5;
+}
+
+.background {
+  height: max-content;
+  width: auto;
+  display: flex;
+  flex-direction: column;
+  background-color: white;
+  align-items: center;
+  flex-wrap: wrap;
+}
+
+.container {
+  width: 327px;
+  height: 474px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  flex-direction: column;
+  margin-bottom: 2rem;
+}
+
+.toolbar ul {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  align-content: center;
+  list-style: none;
+}
+
+.toolbar li {
+  padding: 0;
+  display: none;
+}
+
+.toolbar a {
+  display: flex;
+  height: 20%;
+  list-style: none;
+  align-items: center;
+  text-decoration: none;
+  margin-right: 60px;
+  float: right;
+  margin-top: 40px;
+  justify-content: space-between;
+  padding: 20px;
+  color: black;
+}
+
+.primary {
+  clear: right;
+  text-align: center;
+  font-weight: 700;
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+  align-items: center;
+  margin-top: 2rem;
+}
+
+span {
+  color: #36b37e;
+}
+
+.headline p {
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 28px;
+  margin: 20px 20px;
+}
+
+.social-media ul {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  list-style: none;
+}
+
+.social-media img {
+  height: 20px;
+  width: 20px;
+  margin: 0 5px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.normal-button {
+  margin-top: 100px;
+}
+
+.header {
+  background-image: url("images/header-illsutration-mobile@2x.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.my-recent-work {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  align-content: center;
+}
+
+.my-work {
+  color: #172b4d;
+  font-family: 'Inter', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 72px;
+  text-align: center;
+  padding-bottom: 50px;
+  margin-top: 70px;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.grid-container {
+  display: grid;
+  gap: 10px;
+  align-items: center;
+  justify-content: center;
+}
+
+.card {
+  margin-top: 24px;
+}
+
+.card-work {
+  background-color: #d0d9d4;
+  width: 300px;
+  height: 252px;
+}
+
+.project-information {
+  border: solid 1px #d0d9d4;
+  border-radius: 5px;
+  display: flex;
+  justify-content: flex-end;
+  width: 300px;
+  height: 474px;
+  flex-direction: column;
+  align-items: center;
+}
```

Comment:
- [ ] I would kindly suggest you to use a responsive values in your CSS rules, like percentages (i.e. width: 80%), instead of pixels for proper accessibility and it will help your website UI responsive  and user friendly.